### PR TITLE
[Enhancement] Show create table and desc table add field comment info for external table

### PR DESCRIPTION
**Pull request by chaixiaoxue**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java [hunk 1]
```diff
@@ -194,6 +195,13 @@ public static List<Column> toFullSchemas(Table nativeTbl) {
         return fullSchema;
     }
 
+    public static String toComment(Map<String, String> tableParams) {
```
Comment (stephen-shelby) [inline fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java:198]: ditto

Comment (stephen-shelby) [inline fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java:198]: ditto

Comment (chaixiaoxue) [reply fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java:198]: done

Comment (chaixiaoxue) [reply fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java:198]: done

File: fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java [hunk 2]
```diff
@@ -388,6 +397,7 @@ public static List<Column> toFullSchemasForHiveTable(Table table) {
                 type = Type.UNKNOWN_TYPE;
             }
             Column column = new Column(fieldSchema.getName(), type, true, comment);
+            Column column = new Column(fieldSchema.getName(), type, true, fieldSchema.getComment());
```
[the remark this answers is not in the document]
Comment (chaixiaoxue) [reply fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java]: done

File: fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java [hunk 3]
```diff
@@ -108,6 +108,13 @@ public static String toTableLocation(StorageDescriptor sd, Map<String, String> t
         return sd.getLocation();
     }
 
+    public static String toComment(Map<String, String> tableParams) {
+        if (tableParams.containsKey("comment")) {
+            return tableParams.get("comment");
```
[the remark this answers is not in the document]
Comment (chaixiaoxue) [reply fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java]: done

**Comment by chaixiaoxue** [comment]
@stephen-shelby  pls review again

**Comment by chaixiaoxue** [comment]
Hi @stephen-shelby  , pls reivew this pr

File: fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java [hunk 4]
```diff
@@ -72,6 +72,7 @@ public static IcebergTable toIcebergTable(Table nativeTbl, String catalogName, S
                 .setResourceName(toResourceName(catalogName, "iceberg"))
                 .setRemoteDbName(remoteDbName)
                 .setRemoteTableName(remoteTableName)
+                .setComment(toComment(nativeTbl.properties()))
```
Comment (stephen-shelby) [inline fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java:75]:
```suggestion
                .setComment(tableParams.getOrDefault("comment", ""))
```

Comment (chaixiaoxue) [reply fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java]: done

**Comment by chaixiaoxue** [comment]
@stephen-shelby  Changed, please review, thanks

**Review by stephen-shelby — DISMISSED** [review]

**Review by HangyuanLiu — DISMISSED** [review]

**Comment by chaixiaoxue** [comment]
@stephen-shelby Can you help me rerun fe ut, I just ran it and it reported a timeout error!

**Comment by stephen-shelby** [comment]
fix fe ut

**Comment by chaixiaoxue** [comment]
Hi@stephen-shelby  @HangyuanLiu  , pls run the workflow. thanks

**Comment by chaixiaoxue** [comment]
Hi @stephen-shelby  @HangyuanLiu  pls run the workflow. thanks

**Comment by chaixiaoxue** [comment]
@stephen-shelby @HangyuanLiu The workflow run is complete, please help megre the code.

**Review by stephen-shelby — APPROVED** [review]

**Review by HangyuanLiu — APPROVED** [review]

**Pull request merged**